Add synergia example diagnostics output

s-sajid-ali opened the pull request.

Commit:
Add synergia example diagnostics output

ax3l commented on README.md:
```diff
@@ -28,6 +28,9 @@ The files `example-2d.tar.gz`, `example-3d.tar.gz` and `example-thetaMode.tar.gz
 The scripts that were used in order to produce these openPMD files from Warp can be found in the directory `scripts/warp`.
 
+## Examples created with the Particle-In-Cell code Synergia
+The file `synergia-fodo-diag.h5` was generated by running the `fodo_cxx` example from the accelerator modeling package [`synergia`](https://github.com/fnalacceleratormodeling/synergia2) (using the `devel3` branch, commit id `81c96e4`). Is is the result of running the example file with the following command line options: `./fodo_cxx macroparticles=4096 gridx=8 gridy=8 gridz=8 turns=25` and renamin ...
```
```suggestion

The file `synergia-fodo-diag.h5` was generated by running the `fodo_cxx` example from the accelerator modeling package [`synergia`](https://github.com/fnalacceleratormodeling/synergia2) (using the `devel3` branch, commit id `81c96e4`). Is is the result of running the example file with the following command line options: `./fodo_cxx macroparticles=4096 gridx=8 gridy=8 gridz=8 turns=25` and renaming the resulting diagnostics output `diag.h5`. 
```

ax3l commented:
@s-sajid-ali Thank you for the PR! I took a look at the file and albeit valid, it does not seem to contain particles. Can you double check?
```console
$ h5ls -r synergia-fodo-diag.h5 
/                        Group
/data                    Group
/data/0                  Group
/data/1                  Group
/data/10                 Group
/data/11                 Group
/data/12                 Group
...
```
```console
$ openpmd-ls synergia-fodo-diag.h5 
openPMD series: synergia-fodo-diag
openPMD standard: 1.1.0
openPMD extensions: 0

data author: unknown
data created: 2024-03-29 13:59:33 -0500
data backend: HDF5
generating machine: unknown
generating software: synergia3 (version: 2023.4.12-81c96e4d6)
generating software dependencies: unknown

number of iterations: 26 (groupBased)
  all iterations: 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 

number of meshes: 0

number of particle species: 0
```

s-sajid-ali commented:
Yes it does not contain any particles, it is the output of custom diagnostics as defined here:
https://github.com/fnalacceleratormodeling/synergia2/blob/1a85586989e545da5277a96131facc12d8b0d952/src/synergia/bunch/diagnostics_full2.cc#L60-L86

This was one of the files that the updater failed on before https://github.com/openPMD/openPMD-updater/pull/10. With the updates there it simply notes that data is not present and only updates the top level attributes.

s-sajid-ali commented:
I can add file containing particles from the same run if that helps set the context for this file.

ax3l commented:
I see. I would call this file something like `_empty.h5` then maybe?

Let's also add a valid Synergia file that is a bit more instructive/useful for people to play with (vis, analysis, tests)?

Commit:
add all files resulting from running fodo_cxx

	modified:   README.md
	deleted:    synergia-fodo-diag.h5
	new file:   synergia-fodo.tar.gz

s-sajid-ali commented:
Updated with more files from running the example. Let me know it looks good now.
